shared/chat/conversation/input-area/user-mention-hud/index.js

```diff
@@ -91,18 +76,29 @@ const hudStyle = {
   backgroundColor: globalColors.white,
 }
 
-const _withProps = ({users, filter, selectedIndex}) => {
-  const fullList = users
+const _withProps = ({users, isTeam, selectedIndex, filter}) => {
+  let fullList = users
     .map((u, i) => ({
       fullName: u.fullName,
       key: u.username,
       username: u.username,
     }))
-    .concat({
-      fullName: 'Everyone in this channel',
-      key: 'channel',
-      username: 'channel',
-    })
+    .concat()
+
+  if (isTeam) {
+    fullList = fullList.concat([
+      {
+        fullName: 'Everyone in this channel',
+        key: 'channel',
+        username: 'channel',
+      },
+      {
+        fullName: 'Online users',
```

Comment:
@cecileboucheron I chose this wording because using `"Everyone online in this channel"` means that both `@here` and `@channel` will show up when typing `@channel` into the input box.

Any thoughts on this wording?

Reply:
@thebearjew No we actually don't have online presence right now. They both do the same thing, i.e they ping everyone who's in the channel. 
So is it possible to do:
* big team: `Everyone in this channel` for both
* small team: `Everyone in the team` for both
* group or 1-1 convos: show none

Reply:
@cecileboucheron thanks! Updated!